internal/install/execution_status.go

```diff
@@ -0,0 +1,89 @@
+package install
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type executionStatus struct {
```

Comment:
Not sure if there is a better name, but this name is confusing with the other type below executionStatusRecipe, particularly since this typoe does have a Recipe element in it, and the other type does not but has it in the name... confusing.

Reply:
👍